### DISCUSSION TITLE
enable memory configurations for sidecar containers

### DISCUSSION THREAD

**Discussion by nabeelsaabna** [renovatebot/renovate#22535] [Suggest an Idea]
### Tell us more.

When running with `binarySource=docker`, I can limit the Renovate container memory but the sidecar container remains without any limits.
I want the ability to limit the memory that sidecar container can use.

[Limit a container’s access to memory](https://docs.docker.com/config/containers/resource_constraints/#limit-a-containers-access-to-memory):
-m or --memory | The maximum amount of memory the container can use. If you set this option, the minimum allowed value is 6m (6 megabytes). That is, you must set the value to at least 6 megabytes.
-- | --

_this can also be applied to CPU and other flags depending on the solution complexity_

**Comment by rarkins** [comment]
Looks good. Should be a global config option (applicable to all sidecar containers), and disabled by default.

**Reply by viceice** [reply]
I think that can be `dockerCliOptions` admin config, which is a string and gets appended to the run command before the docker image name.
